ocs_ci/ocs/constants.py

```diff
@@ -718,6 +719,8 @@
 CEPHFS_CTRLPLUGIN_LABEL = "app=openshift-storage.cephfs.csi.ceph.com-ctrlplugin"
 RBD_CTRLPLUGIN_LABEL = "app=openshift-storage.rbd.csi.ceph.com-ctrlplugin"
 PROMETHEUS_POD_LABEL = "app.kubernetes.io/name=prometheus"
+CEPHFS_CTRLPLUGIN_NAME = "openshift-storage.cephfs.csi.ceph.com-ctrlplugin"
+RBD_CTRLPLUGIN_NAME = "openshift-storage.rbd.csi.ceph.com-ctrlplugin"
```

Comment:
@paraggit The pod renaming seems to be addressed in this PR https://github.com/red-hat-storage/ocs-ci/pull/11807/files. Maybe this could be incorporated.

Reply:
Yes right , I have removed those variables